packages/core/src/components/image-overlay/image-overlay.scss

```diff
@@ -0,0 +1,74 @@
+@import "~@dso-toolkit/sources/src/styling/variables";
+
+@import "../../icon/di";
+
+@import "~@dso-toolkit/sources/src/styling/global/button";
+@import "~@dso-toolkit/sources/src/styling/mixins/box-sizing";
+@import "~@dso-toolkit/sources/src/styling/mixins/sr-only";
+
+@import "~@dso-toolkit/sources/src/components/button/button";
+
+$dso-image-overlay-z-index: 20;
+
+:host {
+  display: inline-block;
+  position: relative;
+}
+
+:host(:hover) .open,
+.open:focus {
+  opacity: 1;
+}
+
+.open {
+  @include button_secondary($icon-only: true);
+
+  border: 0;
+  box-shadow: 0 1px 4px 0 rgba(0, 0, 0, 0.5);
+  min-width: auto;
+  padding: $u1;
+  opacity: 0;
+  position: absolute;
+  right: $u2;
+  top: $u2;
+}
+
+.dimmer {
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  z-index: 1;
+  display: flex;
+  padding: $u4;
+  background-color: fade-out($wit, 0.2);
+  justify-content: center;
+}
+
+.wrapper {
+  box-shadow: 0 8px 24px 0 rgba(25, 25, 25, 0.4);
+  position: relative;
+  z-index: $dso-image-overlay-z-index;
```

Comment:
@daverhelst Let op: Maandag wordt dit PR gemerged en heb jij 'n extra item voor in je register